README.md

```diff
@@ -14,15 +14,13 @@ will get you up and running in no time!
 
 There may need to be modifications to the included `docker-compose.yml` file to accommodate your needs or your environment. These instructions are a guideline for what you should generally do.
 
-1. `mkdir -p data/{sentry,postgres}` - Make our local database and sentry config directories.
-    This directory is bind-mounted with postgres so you don't lose state!
-2. `docker-compose build` - Build and tag the Docker services
-3. `docker-compose run --rm web config generate-secret-key` - Generate a secret key.
-    Add it to `docker-compose.yml` in `base` as `SENTRY_SECRET_KEY`.
-4. `docker-compose run --rm web upgrade` - Build the database.
+1. `docker-compose build` - Build and tag the Docker services
+2. `docker-compose run --rm web config generate-secret-key` - Generate a secret key.
+    Add it to `docker-compose.yml` in `x-defaults` as `SENTRY_SECRET_KEY`.
+3. `docker-compose run --rm web upgrade` - Build the database.
     Use the interactive prompts to create a user account.
-5. `docker-compose up -d` - Lift all services (detached/background mode).
-6. Access your instance at `localhost:9000`!
+4. `docker-compose up -d` - Lift all services (detached/background mode).
+5. Access your instance at `0.0.0.0:9000`!
```

Comment:
Why this change? I disagree with this one. It doesn’t even make sense to use 0.0.0.0 from the client. I understand it works, but why not localhost anymore? 0.0.0.0 is unlikely to work with everything.

Reply:
To make clear that IP address may be external as well. May be use `IP_ADDRESS:9000` instead. localhost led me astray. Any way I can get `localhost` back.

Reply:
i suggest `http://127.0.0.1:9000` which is clickable :)